contracts/Market.sol

```diff
@@ -2,6 +2,8 @@
 pragma solidity =0.8.15;
 import "./Certificate.sol";
 import "./RestrictedNORI.sol";
+import "./PausableAccessPreset.sol";
```

Comment:
adding imports where I find them missing because we discovered that's why the IDE has trouble with cmd-click: it only works if the relevant file is explicitly imported